source/particle/property/interface.cc

```diff
@@ -393,6 +393,40 @@ namespace aspect
         return (update & (update_default | update_values | update_gradients));
       }
 
+      template <int dim>
+      bool
+      Manager<dim>::plugin_name_exists(const std::string &name) const
+      {
+        return (std::find(plugin_names.begin(),plugin_names.end(),name) != plugin_names.end());
+      }
+
+      template <int dim>
```

Comment:
```suggestion


      template <int dim>
```

Reply:
I thought the convention was one new line, but I now see that the most of this file is not consistent with the amount of new lines being used. Is two lines the convention?

Reply:
Three empty lines between function definitions.

Reply:
So should I also add 3 lines between all the other function definitions in this file, or am I missing something?

Reply:
In principle yes, but if it's for lines that you didn't touch in this patch, then let's not worry about it here. (But if you wanted to fix existing code in a separate PR, that is always welcome.)

Reply:
makes sense. I will make another pull request after this one to fix that.